enterprise/server/remote_execution/workspace/workspace.go

```diff
@@ -551,6 +551,21 @@ func (ws *Workspace) Clean() error {
 			// from the inputs index (this should be pretty uncommon).
 			delete(ws.Inputs, outputDirPath)
 		}
+		for _, outputPath := range cmd.GetOutputPaths() {
```

Comment:
Is it worth combining the cmd.GetOutputPaths() and cmd.GetOutputDirectories() slices? The logic appears to be identical.

Reply:
Done. Combined the output_files logic as well, since I'm pretty sure fixing the TODO in func would result in all 3 loop bodies being identical, and also output_files is probably only used by older clients anyway.